plugins/quickicon/eos/src/Extension/Eos.php

```diff
@@ -34,12 +34,12 @@ final class Eos extends CMSPlugin implements SubscriberInterface
     use DatabaseAwareTrait;
 
     /**
-     * The EOS date for 4.4.
+     * The EOS date for 5.x.
```

Comment:
```suggestion
     * The EOS date for 5.x
```

Reply:
The dot is also there in 4.4-dev, so if we change it we should also change it in 4.4-dev. But I don't really see the need to change it at all.

Reply:
All other variable description end with a dot, too, so I think you are wrong.

Reply:
> All other variable descriptions in that file end with a dot, too, so I think you are wrong.

You are correct, I now see that the dot is meant to indicate the end of a line.

Reply:
I would suggest to remove the version number.

```suggestion
     * The EOS date for the current major version.
```